pootle/apps/pootle_store/views.py

```diff
@@ -350,63 +366,105 @@ def get_units(request):
     except Resolver404:
         raise Http404('Unrecognised path')
 
-    units_qs = Unit.objects.get_translatable(
-        user=request.profile,
-        **path_kwargs)
-    units_qs = units_qs.order_by("store", "index")
+    related_selects = (
```

Comment:
in this case i want to keep - it improves formatting - and also takes this list out of the function call